.devcontainer/devcontainer_setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# initializing conda in the current shell session
+eval "$(conda shell.bash hook)"
+
+conda create -n pyrit-dev python=3.11 -y
+conda activate pyrit-dev
+conda install ipykernel -y
+pip install -e ."[dev,playwright]"
```

Comment:
```suggestion
pip install -e ."[dev,all]"
```
We'd probably want `all` here so that all the functional extras are added. `all` = all extras except `dev` which includes only the ones needed for dev, testing etc.